src/OrchardCore.Modules/OrchardCore.Liquid/Startup.cs

```diff
@@ -106,3 +113,32 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddLiquidFilter<ShortcodeFilter>("shortcode");
     }
 }
+
+[RequireFeatures("OrchardCore.Resources")]
+public sealed class ResourcesStartup : StartupBase
+{
+    public override void ConfigureServices(IServiceCollection services)
+    {
+        services.AddLiquidFilter<AppendVersionFilter>("append_version")
+            .AddLiquidFilter<ResourceUrlFilter>("resource_url");
+    }
+}
+
+[RequireFeatures("OrchardCore.Html")]
+public sealed class HtmlStartup : StartupBase
+{
+    public override void ConfigureServices(IServiceCollection services)
+        => services.AddLiquidFilter<SanitizeHtmlFilter>("sanitize_html");
```

Comment:
Don't use online function

Reply:
I though you would comment on this one :)